libraries/botbuilder-lg/tests/templateEngineThrowException.test.js

```diff
@@ -6,48 +6,49 @@ function GetExampleFilePath(fileName){
 }
 
 const StaticCheckExceptionData  = [
-    "ErrorTemplateParameters.lg",
-    "NoNormalTemplateBody.lg",
-    "ConditionFormatError.lg",
-    "NoTemplateRef.lg",
-    "TemplateParamsNotMatchArgsNum.lg",
-    "ErrorSeperateChar.lg",
-    "ErrorSeperateChar2.lg",
-    "MultilineVariation.lg",
-    "InvalidTemplateName.lg",
-    "InvalidTemplateName2.lg",
-    "DuplicatedTemplates.lg",
-    "LgTemplateFunctionError.lg",
-    "SwitchCaseFormatError.lg",
-    "InvalidLGFileImportPath.lg",
-    "DuplicatedTemplatesInImportFiles.lg",
-    "ErrorStructuredTemplate.lg"
-    ];
+    'MultilineVariation.lg',
+    'ErrorTemplateParameters.lg',
+    'NoNormalTemplateBody.lg',
+    'ConditionFormatError.lg',
+    'NoTemplateRef.lg',
+    'TemplateParamsNotMatchArgsNum.lg',
+    'ErrorSeperateChar.lg',
+    'ErrorSeperateChar2.lg',
+    'MultilineVariation.lg',
+    'InvalidTemplateName.lg',
+    'InvalidTemplateName2.lg',
+    'DuplicatedTemplates.lg',
+    'LgTemplateFunctionError.lg',
+    'SwitchCaseFormatError.lg',
+    'InvalidLGFileImportPath.lg',
+    'DuplicatedTemplatesInImportFiles.lg',
+    'ErrorStructuredTemplate.lg'
```

Comment:
These seem like constants.

Reply:
yes, These correspond to some LG test files